app/src/main/java/org/mozilla/tv/firefox/fxa/FxaRepo.kt

```diff
@@ -26,6 +26,12 @@ import org.mozilla.tv.firefox.fxa.FxaRepo.AccountState.NOT_AUTHENTICATED
 
 private const val LOGTAG = "FxaRepo"
 
+private val APPLICATION_SCOPES = arrayOf(
+    // We don't use sync, however, if we later add the sync scope, already authenticated users
+    // will have to log in again. To avoid this in the future, we preemptively declare a sync scope.
+    "https://identity.mozilla.com/apps/oldsync"
```

Comment:
Are there potentially any other scopes that we might want to use in the future?  Is there anyone we should ask about this?